Description: (Debian packaging) Fix lintian warning by using local mathjax package instead of CDN
Author: Kyle Kloberdanz <kyle.kloberdanz@mongodb.com>
Forwarded: not-needed
--- a/src/libmongoc/doc/conf.py
+++ b/src/libmongoc/doc/conf.py
@@ -30,4 +30,6 @@ ]

+mathjax_path = "file:///usr/share/javascript/mathjax/MathJax.js"
+
 # General information about the project.
 project = "libmongoc"
 copyright = "2017-present, MongoDB, Inc"
